diff -aurN qjackctl-0.1.0.orig/configure.in qjackctl-0.1.0/configure.in
--- qjackctl-0.1.0.orig/configure.in	2003-10-19 23:47:11.000000000 +0200
+++ qjackctl-0.1.0/configure.in	2003-10-24 20:55:53.930225000 +0200
@@ -47,8 +47,8 @@
 fi
 AC_SUBST(ac_qtlib)
 
-CFLAGS="$CFLAGS -I$QTDIR/include"
-CPPFLAGS="$CPPFLAGS -I$QTDIR/include"
+CFLAGS="$CFLAGS -I$QTDIR/include/qt"
+CPPFLAGS="$CPPFLAGS -I$QTDIR/include/qt"
 LIBS="-L$QTDIR/lib -L/usr/X11R6/lib"
 
 AC_CACHE_CHECK([for Qt library version >= 3.1.1],
diff -aurN qjackctl-0.1.0.orig/Makefile.in qjackctl-0.1.0/Makefile.in
--- qjackctl-0.1.0.orig/Makefile.in	2003-10-15 23:09:57.000000000 +0200
+++ qjackctl-0.1.0/Makefile.in	2003-10-24 20:51:46.773798488 +0200
@@ -43,8 +43,8 @@
 	@make -f $(name).mak $(name)
 
 install:	$(name) icons/$(name).xpm
-	@install -vD $(name) $(prefix)/bin/$(name)
-	@install -vD icons/$(name).xpm $(prefix)/share/icons/$(name).xpm
+	@install -vD $(name) $(DESTDIR)$(prefix)/bin/$(name)
+	@install -vD icons/$(name).xpm $(DESTDIR)$(prefix)/share/pixmaps/$(name).xpm
 
 uninstall:	$(prefix)/bin/$(name)
 	@rm -vf $(prefix)/bin/$(name)
