--- qjackctl-0.3.7/Makefile.in.orig	2010-08-23 18:27:45.000000000 +0200
+++ qjackctl-0.3.7/Makefile.in	2011-04-18 12:20:53.506064376 +0200
@@ -103,7 +103,12 @@
 	@make INSTALL_ROOT=$(DESTDIR) -f $(name).mak install
 	@install -d -v -m 0755 $(DESTDIR)$(prefix)/share/locale
 	@install -d -v -m 0755 $(DESTDIR)$(prefix)/share/man/man1
-	@install -v -m 0644 $(translations_targets) $(DESTDIR)$(prefix)/share/locale
+	@for f in $(translations_targets) ; do \
+		ff=`basename $$f .qm` ; \
+		ll=$${ff##*_} ; \
+		install -d -v -m 0755 $(DESTDIR)$(prefix)/share/locale/$$ll/LC_MESSAGES ; \
+		install -v -m 0644 $$f $(DESTDIR)$(prefix)/share/locale/$$ll/LC_MESSAGES/qjackctl.qm ; \
+	done
 	@install -v -m 0644 $(name).1 $(DESTDIR)$(prefix)/share/man/man1/$(name).1
 
 uninstall:	$(DESTDIR)$(prefix)/bin/$(name)
